src/Proto.Cluster/PidCache.cs

```diff
@@ -23,7 +23,7 @@ internal static class PidCache
 
         internal static void Setup()
         {
-            var props = Actor.FromProducer(() => new PidCacheWatcher());
+            var props = Actor.FromProducer(() => new PidCacheWatcher()).WithGuardianSupervisorStrategy(Supervision.KeepAliveStrategy);

```

Comment:
IMO, AlwaysRestartStrategy would be a more descriptive name, as "KeepAlive" could mean anything, e.g. ignore errors etc.

Reply:
Yep, just got it updated.